### PR TITLE
Ignore older than and commit time.

### PR DESCRIPTION
**Pull request by ptr07**
Add ignore-older-than config option. 
Commit time added to CommitWrapper and available in template.

### CODE REVIEW

File: src/main/java/info/plichta/maven/plugins/changelog/model/CommitWrapper.java [hunk 1]
```diff
@@ -31,6 +29,12 @@
  * Model class representing one GIT commit.
  */
 public class CommitWrapper {
+
```
Comment (jakubplichta) [inline src/main/java/info/plichta/maven/plugins/changelog/model/CommitWrapper.java]: please remove unnecessary empty lines (i.e. use only one)

File: src/main/java/info/plichta/maven/plugins/changelog/model/CommitWrapper.java [hunk 2]
```diff
@@ -71,4 +75,8 @@ public String getCommitLink() {
     public Map<String, Object> getExtensions() {
         return extensions;
     }
+
+    public String getCommitTime() {
```
Comment (jakubplichta) [inline src/main/java/info/plichta/maven/plugins/changelog/model/CommitWrapper.java]: please use Java 8 date/time API

File: src/main/java/info/plichta/maven/plugins/changelog/CommitFilter.java [hunk 3]
```diff
@@ -19,22 +19,38 @@
...
 
+
         final Predicate<RevCommit> includePred =
                 revCommit -> includePattern.matcher(revCommit.getFullMessage()).matches();
         final Predicate<RevCommit> excludePred =
                 revCommit -> excludePattern.map(p -> p.matcher(revCommit.getFullMessage()).matches()).orElse(false);
-        predicate = includePred.and(excludePred.negate());
+        final int maxTime = getMinimumCommitTime(ignoreOlderThen);
+        final Predicate<RevCommit> timePred = revCommit -> revCommit.getCommitTime() >= maxTime;
+
```
Comment (jakubplichta) [inline src/main/java/info/plichta/maven/plugins/changelog/CommitFilter.java]: remove unnecessary empty lines

File: src/main/java/info/plichta/maven/plugins/changelog/ChangeLogMojo.java [hunk 4]
```diff
@@ -78,6 +79,9 @@
     @Parameter(property = "project.artifactId")
     private String tagPrefix;
 
+    @Parameter
+    private Date ignoreOlderThen;
```
Comment (jakubplichta) [inline src/main/java/info/plichta/maven/plugins/changelog/ChangeLogMojo.java]: use Zoned/LocalDateTime

**Comment by ptr07** [comment]
Tkanks for review. Empty lines removed. Date types migrated to java8 API.

**Pull request merged**